app/core/store/modules/teacherDashboard.js

```diff
@@ -417,9 +423,12 @@ export default {
           project: (options.data || {}).levelSessions
         }
         fetchPromises.push(dispatch('levelSessions/fetchForClassroomMembers', { classroom, options: levelSessionOptions }, { root: true }))
+        fetchPromises.push(dispatch('aiProjects/fetchForClassroomMembers', { classroom }, { root: true }))
```

Comment:
I'm guessing the architecture is like this where we fetch all even if the course selected is not HS, correct? 
Something we should optimize later? We should create a list of such things to tackle later.